Plots/Satellites/NCL_sat_2.py

```diff
@@ -0,0 +1,436 @@
+"""
+NCL_sat_2.py
+===============
+This script illustrates the following concepts:
+   - unpacking 'short' data
+   - Drawing filled contours over a satellite map
+   - Explicitly setting contour fill colors
+   - Finding local high pressure values
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/sat_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/sat_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import xarray as xr
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import numpy as np
+from sklearn.cluster import KMeans
+import matplotlib.pyplot as plt
+from matplotlib import colors
+import matplotlib.ticker as mticker
+
+import geocat.datafiles as gdf
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and
+# load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/slp.1963.nc"), decode_times=False)
+
+# Get data from the 24th timestep
+pressure = ds.slp[21, :, :]
```

Comment:
This comment says the 24th time step but you're using what looks like the 21st time step. Can you change the comment or the code here so they match?